### PR TITLE
fix(react-scheduler): set Scheduler's height to Root instead of body

### PR DESCRIPTION
**Pull request by AryamnovEugeniy**
BREAKING CHANGE:

`DayView` plugin's, `WeekView` plugin's and `MonthView` plugin's layout component now doesn't have the `height` property. `height` is now `Scheduler` plugin's root component property.

```diff
...
<DayView
  layoutComponent={({
-   height,
     ...restProps
  }) => (
    <DayView.Layout
-      height={height}
       {...restProps}
    />
  )}
/>
...
<WeekView
  layoutComponent={({
-   height,
     ...restProps
  }) => (
    <WeekView.Layout
-      height={height}
       {...restProps}
    />
  )}
/>
...
<MonthView
  layoutComponent={({
-   height,
     ...restProps
  }) => (
    <MonthView.Layout
-      height={height}
       {...restProps}
    />
  )}
/>
...
<Scheduler
  rootComponent={({
+   height,
     ...restProps
  }) => (
    <Scheduler.Root
+      height={height}
       {...restProps}
    />
  )}
/>
...
```

### CODE REVIEW

File: packages/dx-react-scheduler-material-ui/src/templates/layouts/horizontal-view-layout.test.jsx [hunk 1]
```diff
@@ -48,16 +47,7 @@ describe('Horizontal View Layout', () => {
...
-  });
-
-  it('should replace style of the root element', () => {
-    const tree = shallow((
-      <HorizontalViewLayout {...defaultProps} style={{ height: 1 }} />
-    ));
-
-    expect(tree.prop('style'))
-      .toMatchObject({ height: 1 });
+      .toMatchObject({ a: 1 });
```
Comment (MaximKudriavtsev) [inline packages/dx-react-scheduler-material-ui/src/templates/layouts/horizontal-view-layout.test.jsx]: We can remove this test

File: packages/dx-react-scheduler-material-ui/src/templates/layouts/vertical-view-layout.test.jsx [hunk 2]
```diff
@@ -50,16 +49,7 @@ describe('Vertical View Layout', () => {
...
-  });
-
-  it('should replace style of the root element', () => {
-    const tree = shallow((
-      <VerticalViewLayout {...defaultProps} style={{ height: 1 }} />
-    ));
-
-    expect(tree.prop('style'))
-      .toMatchObject({ height: 1 });
+      .toMatchObject({ a: 1 });
```
Comment (MaximKudriavtsev) [inline packages/dx-react-scheduler-material-ui/src/templates/layouts/vertical-view-layout.test.jsx]: Same

File: packages/dx-react-scheduler/api/dx-react-scheduler.api.md [hunk 3]
```diff
@@ -480,6 +479,7 @@ export const Scheduler: React.ComponentType<SchedulerProps>;
 export namespace Scheduler {
     export interface RootProps {
         children?: React.ReactNode;
+        height?: number | 'auto';
```
Comment (MaximKudriavtsev) [inline packages/dx-react-scheduler/api/dx-react-scheduler.api.md]: The height property is required

File: packages/dx-react-scheduler/src/plugins/scheduler-core.tsx [hunk 4]
```diff
@@ -20,9 +20,9 @@ class SchedulerCoreBase extends React.PureComponent<SchedulerProps> {
       >
         <Getter name="appointments" value={appointments(data)} />
         <Getter name="formatDate" value={formatDateTimeGetter(locale)} />
-        <Getter name="layoutHeight" value={height} />
         <Template name="root">
-          <Root>
+          <Root height={height} >
+            <TemplatePlaceholder name="drawer" />
```
Comment (MaximKudriavtsev) [inline packages/dx-react-scheduler/src/plugins/scheduler-core.tsx]: We should use only common words. `Drawer` -> schedulerContent / schedulerRoot or something else

File: packages/dx-react-scheduler/src/types/scheduler/scheduler.types.ts [hunk 5]
```diff
@@ -6,6 +6,8 @@ export namespace Scheduler {
   export interface RootProps {
     /** A React node to be placed in the root layout. */
     children?: React.ReactNode;
+    /** The Scheduler's height */
+    height?: number | 'auto';
```
Comment (MaximKudriavtsev) [inline packages/dx-react-scheduler/src/types/scheduler/scheduler.types.ts]: The `height` is required

**Comment by ushkal** [comment]
Can we provide the `height` option at the Scheduler level?

**Pull request merged**